ksqldb-engine/src/main/java/io/confluent/ksql/function/UdafLoader.java

```diff
@@ -33,15 +33,15 @@
 /**
  * Loads user defined aggregate functions (UDAFs)
  */
-class UdafLoader {
+public class UdafLoader {
```

Comment:
Hmm... does this have to be made public?  (I'm not sure if there's some good reason why it would need to stay protected...)